### PR TITLE
Iter9

### PR DESCRIPTION
**Pull request by justEngineer**

### CODE REVIEW

File: internal/http/server/server_handler.go [hunk 1]
```diff
@@ -109,36 +105,129 @@ func (h *Handler) UpdateMetric(w http.ResponseWriter, r *http.Request) {
...
+	}
+	if len(urlPart[1]) != 0 {
+		http.Error(w, "Wrong URL", http.StatusBadRequest)
+		return
+	}
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	tmplFile := filepath.Join(filepath.Dir(wd), h.config.PathToHTMLTemplate)
```
Comment (arttet) [inline internal/http/server/server_handler.go:122]: CHANGE REQUEST: it won't work if you move it into a Docker container. You need to embed this file.

File: internal/logger/logger.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+func (r *loggingResponseWriter) WriteHeader(statusCode int) {
+	// записываем код статуса, используя оригинальный http.ResponseWriter
+	r.ResponseWriter.WriteHeader(statusCode)
+	r.responseData.status = statusCode // захватываем код статуса
+}
+
+// Log будет доступен всему коду как синглтон.
+// Никакой код навыка, кроме функции InitLogger, не должен модифицировать эту переменную.
+// По умолчанию установлен no-op-логер, который не выводит никаких сообщений.
+var Log *zap.Logger = zap.NewNop()
```
Comment (arttet) [inline internal/logger/logger.go:41]: CHANGE REQUEST: please don't use global variables.

**Review by arttet — CHANGES_REQUESTED** [review]

File: internal/http/server/server_handler.go (hunk 1 above)
Comment (justEngineer) [reply internal/http/server/server_handler.go]: done

File: internal/logger/logger.go (hunk 2 above)
Comment (justEngineer) [reply internal/logger/logger.go]: done

**Review by arttet — APPROVED** [review]

**Pull request merged**